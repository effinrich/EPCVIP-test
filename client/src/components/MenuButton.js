import React from 'react'
import PropTypes from 'prop-types'

import { Box, Text } from 'grommet'
import RoutedButton from './RoutedButton'

const MenuButton = ({ label, ...rest }) => {
  return (
    <RoutedButton hoverIndicator={{ color: 'control' }} {...rest}>
      <Box pad="small" gap="xsmall" justify="center">
        <Text>{label}</Text>
      </Box>
    </RoutedButton>
  )
}

MenuButton.propTypes = {
  label: PropTypes.string
}

export default MenuButton
