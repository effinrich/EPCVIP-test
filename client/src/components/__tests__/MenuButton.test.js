import React from 'react'

import { render } from 'utils/testUtils/helperRtl'

import MenuButton from '../MenuButton'

describe('MenuButton Component', () => {
  it('should render without crashing', () => {
    const { container } = render(<MenuButton />)
    expect(container).toMatchSnapshot()
  })

  it('should render child element', () => {
    const { container } = render(<MenuButton />)
    expect(container.firstChild).toBeDefined()
  })
})
