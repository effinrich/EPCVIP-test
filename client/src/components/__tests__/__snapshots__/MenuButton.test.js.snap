// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MenuButton Component should render without crashing 1`] = `
<div>
  <button
    class="StyledButton-sc-323bzc-0 dtwdYB"
    type="button"
  >
    <div
      class="StyledBox-sc-13pk1d4-0 kBTLoi"
    >
      <span
        class="StyledText-sc-1sadyjn-0 DkZvS"
      />
    </div>
  </button>
</div>
`;
